core/templates/pages/story-viewer-page/story-viewer-page.directive.ts

```diff
@@ -290,30 +171,13 @@ angular.module('oppia').directive('storyViewerPage', [
               }
             );
 
-            ctrl.explorationCardIsShown = false;
             // The pathIconParameters is an array containing the co-ordinates,
             // background color and icon url for the icons generated on the
             // path.
             ctrl.pathIconParameters = [];
-            ctrl.activeHighlightedIconIndex = -1;
-            ctrl.MIN_HEIGHT_FOR_PATH_SVG_PX = 220;
-            ctrl.ODD_SVG_HEIGHT_OFFSET_PX = 150;
-            ctrl.EVEN_SVG_HEIGHT_OFFSET_PX = 280;
-            ctrl.ICON_Y_INITIAL_PX = 35;
-            ctrl.ICON_Y_INCREMENT_PX = 110;
-            ctrl.ICON_X_MIDDLE_PX = 225;
-            ctrl.ICON_X_LEFT_PX = 55;
-            ctrl.ICON_X_RIGHT_PX = 410;
-            ctrl.svgHeight = ctrl.MIN_HEIGHT_FOR_PATH_SVG_PX;
             ctrl.nextExplorationId = null;
 
             $anchorScroll.yOffset = -80;
```

Comment:
Just to check, is this still relevant?

Reply:
No longer relevant - all of this had to do with curved path svg, which has been removed

Reply:
Done.

Reply:
Hm, can you please remove all the unused code in this file? Thanks!

Reply:
Done.